all/build.gradle

```diff
@@ -31,7 +31,7 @@ for (subproject in rootProject.subprojects) {
 }
 
 dependencies {
-    compile subprojects.minus(project(':grpc-protobuf-lite'))
+    implementation subprojects.minus(project(':grpc-protobuf-lite'))
```

Comment:
It seems we should use `api` here. @ejona86 what do you think?

Reply:
It is probably better to use `api` to avoid breaking people. That said, I don't have too much sympathy for someone using grpc-all these days (vs early-on-pre-1.0 when it was used more heavily).